tests/unit_tests/test_filters.py

```diff
@@ -241,8 +242,14 @@ def test_first_moment(run_in_tmpdir, box_model):
         assert first_score(sph_flux_tally) == approx(flux)
         assert first_score(zernike_tally) == approx(scatter)
 
-
 def test_energy():
     f = openmc.EnergyFilter.from_group_structure('CCFE-709')
     assert f.bins.shape == (709, 2)
     assert len(f.values) == 710
+
+def test_lethargy_bin_width():
```

Comment:
PEP8: [Surround top-level function and class definitions with two blank lines](https://peps.python.org/pep-0008/#blank-lines)